clusterloader2/pkg/tuningset/randomized_time_limited.go

```diff
@@ -34,15 +35,34 @@ func newRandomizedTimeLimitedLoad(params *api.RandomizedTimeLimitedLoad) TuningS
 	}
 }
 
+type scheduleEntry struct {
+	action    func()
+	startTime time.Time
+}
+
 func (r *randomizedTimeLimitedLoad) Execute(actions []func()) {
-	var wg wait.Group
+	var schedule []scheduleEntry
```

Comment:
Why? What's the point of changing it?

Reply:
Reducing number of gorouitnes (and memory) used.
TBH, I wanted to introduced this changed due to pod store creation flake. In theory goroutines shouldn't be the bottleneck. However if we can reduce something that potentially can be a limit, I think that's something we want to do.